docs/pkgdown.yml

```diff
@@ -19,8 +19,8 @@ articles:
   queries_dataset: queries_dataset.html
   questionnaires: questionnaires.html
   visits_periods: visits_periods.html
-last_built: 2023-07-19T10:42Z
+last_built: 2023-08-03T06:39Z
 urls:
-  reference: https://pharmaverse.github.io/admiral/cran-release/reference/
-  article: https://pharmaverse.github.io/admiral/cran-release/articles/
+  reference: https://pharmaverse.github.io/admiral/reference
```

Comment:
Think we need to keep the `"/cran-release/"` parts

Reply:
Not sure how that happened!